distributed/scheduler.py

```diff
@@ -1328,6 +1382,13 @@ def validate(self):
 
                 pdb.set_trace()
 
+    def get_nbytes_deps(self):
+        nbytes: Py_ssize_t = 0
+        ts: TaskState
+        for ts in self._dependencies:
+            nbytes += ts.get_nbytes()
+        return nbytes
```

Comment:
If this is used only once then I wouldn't mind avoiding the function definition and moving this code to the call site.  Thoughts?

Reply:
We are Cythonizing this function as it is in `scheduler.py`, which would not be the case if it lived in `stealing.py`. Of course we could consider Cythonizing `stealing.py`, but for one line of code I'm not sure it is worth it.

Reply:
OK, sounds good.  Thanks for the explanation.